### ISSUE TITLE
new term for specific_enthalpy

### ISSUE THREAD

**Issue by oceannetworkscanada2** [cf-convention/cf-conventions#311]
Ocean Networks Canada deploys Lufft SF501 weather stations at many locations on the west coast of British Columbia. One of the outputs from these types of instruments is specific enthalpy described as in the manual: 

Parameter of state of the humid air, composed of the specific enthalpies (heat capacity) of
the components of the mixture and related to the mass fraction of the dry air (at 0°C).

Would like to request a term for specific_enthalpy. Is there a need for this term? Is it possible to get it added to the CF Standard Name vocabulary?

**Comment by JonathanGregory** [comment]
Yes, specific enthalpy is an appropriate quantity to describe with a standard name. We already have various enthalpy quantities, but not this one. I would call it `specific_enthalpy_of_air` (J kg-1), like e.g. `specific_kinetic_energy_of_air`, which is in the table.

**Comment by oceannetworkscanada2** [comment]
great, that is good news.

I am new to the process. What are the next steps to ask the community to add this term to the vocabulary? Once there is a thread in this discussion is that enough to get the process started?

**Comment by JonathanGregory** [comment]
Yes, starting the thread, as you have done, begins the process. If you could write out explicitly your proposal for a new standard name, with canonical units and a definition modelled on other related ones in the table, that would be perfect. Others may comment, and at some point the manager of standard names will summarise. Jonathan

**Comment by oceannetworkscanada2** [comment]
thank you Jonathon.

cfstandardname: specific_enthalpy_of_air
canonical units: J Kg-1
definition: the enthalpy of air per unit of mass, composed of the specific enthalpies of the components of the mixture and related to the mass fraction of the dry air (at 0°C)

**Comment by taylor13** [comment]
"and related to the mass fraction of the dry air" seems vague to me.  I understand that the enthalpy of air will depend on the relative amounts of various constituents (e.g., N2, O2, and trace species), which would often be assumed to be fixed, and that it also depends on the water vapor amount, which would normally not be considered fixed.  Is the definition saying that only the enthalpy of dry air is being considered, or is this the enthalpy of the mixture.  Perhaps the following would be clearer:
definition: the enthalpy of air per unit mass, which can often be approximated by a mass-weighted mean of the enthalpies of dry air and of the water vapor in the air.
Please check that this statement is actually correct, as I have written it down from memory.

**Comment by oceannetworkscanada2** [comment]
Taylor13, 

I agree that the statement 'related to the mass fraction of the dry air' is somewhat vague. I admit I was using the definition as given in the instrument manual for the Lufft WS501 weather station that we use in our installations. 

I think that your statement is correct above, but I wanted to share one more definition:

definition: Specific enthalpy (enthalpy per kg of dry air) of moist air is the sum of the specific enthalpy of dry air and the specific enthalpy of  the water vapor in the air. I wonder if we should use 'sum of the enthalpies of dry air and water vapor in the air'  instead of 'approximated by a mass-weighted mean'?

For completeness, here is the equation for Specific enthalpy:

h = ha + H*hg; 

where ha is the specific enthalpy of dry air, H is the humidity ratio, and hg is the specific enthalpy of water vapor.

ha, This is given by the equation cpa*T; where cpa is the specific heat of air, and T is the temperature of air.
You have to memorize the value of cpa which is 1.006 kJ/kgC since this is constant. T must be given in the problem, or it can simply be measured with a thermometer or other temperature measuring devices.

H, Humidity ratio is the ratio of the mass of water vapor in the moist air and the mass of dry air. In layman's terms, it gives the wetness of air. Along with the temperature of air, this must be given in the problem.

Hg. Using a steam table, this can easily be found. Once you have the temperature, bring out your saturated steam table and look for the value of hg. This value has the units kJ/kg.

**Comment by taylor13** [comment]
My proposed definition ("weighted-mean of ...") could easily be misinterpreted in a way that in computing the mean, specific humidity would be used rather than the water vapor mixing ratio (referred to as "humidity ratio" in the above).   So a second attempt:

definition: the enthalpy of air per unit mass, which can be computed for an air sample as the sum of the enthalpy of the dry air and the enthalpy of the water water vapor in that air, divided by the mass of dry air.

**Comment by oceannetworkscanada2** [comment]
taylor13

Thank you. I am in support of your definition above:

definition: the enthalpy of air per unit mass, which can be computed for an air sample as the sum of the enthalpy of the dry air and the enthalpy of the water water vapor in that air, divided by the mass of dry air.

**Comment by oceannetworkscanada2** [comment]
Ok now pending formal acceptance into the vocabulary. 
Hopefully this can be reviewed for the next version.

Cheers.

**Comment by ChantelRidsdale** [comment]
Hello @JonathanGregory and @taylor13, 

@oceannetworkscanada2 has left ONC, and I have inherited this task. Thank you both for your help in the above string! I'm just getting up to speed, and have never undertaken this type of task before. I see that this issue has been included in discussion#114, and that issue was closed today. I see that the workshop took place in September, does that mean the term has been accepted? No news it good news? Is there anything else to do? Is there somewhere we can check it's been included in the list, is there a link available?

**Comment by JonathanGregory** [comment]
I don't see this proposal in the [status list](http://cfeditor.ceda.ac.uk/proposals/1?status=active&namefilter=&proposerfilter=&descfilter=&filter+and+display=filter). I wonder whether it has been overlooked. I wonder whether Alison @japamment can advise. Jonathan

**Comment by feggleton** [comment]
Hi all, apologies this has been overlooked as it's in the wrong repo. 

@ChantelRidsdale could you please add your request to https://github.com/cf-convention/discuss/issues 'discuss' repo. There is an issue template called standard name request which should also explain how to format your request and the rules. This will then be discussed and once it is published you will be informed and it will show here: http://cfconventions.org/Data/cf-standard-names/80/build/cf-standard-name-table.html. Apologies for the delay in explaining the process to you.
